src/platform_impl/macos/window_delegate.rs

```diff
@@ -275,12 +287,8 @@ extern "C" fn window_did_move(this: &Object, _: Sel, _: id) {
 extern "C" fn window_did_change_screen(this: &Object, _: Sel, _: id) {
```

Comment:
We can get rid of this callback. Tracking this in `windowDidChangeScreen` isn't necessary because `windowDidChangeBackingProperties` will be invoked if the backing scale factor changes.